runelite-client/src/main/java/net/runelite/client/plugins/screenmarkers/ScreenMarkerOverlay.java

```diff
@@ -49,6 +50,7 @@
 		setLayer(OverlayLayer.ALWAYS_ON_TOP);
 		setPriority(OverlayPriority.HIGH);
 		setResizable(true);
+		setMinimumSize(new Dimension(OverlayRenderer.OVERLAY_RESIZE_TOLERANCE * 2 + 1, OverlayRenderer.OVERLAY_RESIZE_TOLERANCE * 2 + 1));
```

Comment:
Why is allowing smaller minimum size useful?

Reply:
This particular change is in response to a user finding that resizing a marker they placed over a spell widget, which is notably smaller than 32x32, caused it to grow much larger and they could not resize it back to its original size.

Reply:
I see, makes sense